scripts/deps.py

```diff
@@ -28,19 +28,19 @@ def load_setup_py_data():
     "test"   : meta_src["test"]["requires"],
 }
 
-spec = []
+specs = []
 for name in sys.argv[1:]:
-    spec += section[name]
+    specs += section[name]
 
 # bare python unpins python version causing upgrade to latest
-if 'python' in spec: spec.remove('python')
+specs = [ spec for spec in specs if spec.split(" ")[0] != "python" ]
 
 # add double quotes to specs for windows, fixes #9065
 if "windows" in platform.platform().lower():
-    spec = ['"{}"'.format(s) for s in spec]
+    specs = ['"{}"'.format(spec) for spec in specs]
```

Comment:
could use f-string now

Reply:
Right. I'm still adjusting to all the new possibilities after a decade of stagnation.